### ISSUE TITLE
Keyboard (Tab key) navigation and focus issues

### ISSUE THREAD

**Issue by RobertoMaurizzi** [primefaces/primevue#1817]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report. Asked to open an issue in Discord
[x] feature request. Can be considered part of the planned improvement for better WCAG support, but there are added weird behaviors with focus.
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=110
```

**CodeSandbox Case (Bug Reports)**
(The CodeSandBox has been created from the one linked from the Calendar component demo at https://www.primefaces.org/primevue/showcase/#/calendar )
https://codesandbox.io/s/hardcore-sunset-v6ow6?file=/src/CalendarDemo.vue 

**Current behavior**
Hi! There's something I'm probably missing regarding keyboard navigation/field focus management: I have a complex form with several input fields, multiselects  and some calendars and I'd like to be able to navigate the form using the keyboard only (tab to jump from field to field, enter to open a dropdown, etc.)

The problems I'm having:
  * I can't see how to operate a MultiSelect's overlay using the keyboard. It opens on [space] but then how can I select the items?
  * Tabbing on the Calendar, the overlay does open "on focus" but after I select the date the focus goes... somewhere. In my app, back to the first field for the form, in the CodePen it goes to the tabs of the browser in Chrome, to the first field in Firefox.

See example CodeSandBox linked above (adapted from latest version of your Calendar codesandbox demo, with Vue 3.1.5 and PrimeVue 3.9.1)

**Expected behavior**
Have a way to navigate the complex form using the keyboard only

**Minimal reproduction of the problem with instructions**
Click on the first field, then tab around. You won't be able to select MultiSelect entries and after you open the Calendar overlay, you won't be able to continue to the following fields unless you click on them (closing the overlay sends the focus on other DOM elements or even outside the page, in Chrome)

**What is the motivation / use case for changing the behavior?**
Allow users to fill loooong forms using only the keyboard, so they can be faster.

**Please tell us about your environment:**

MacOS Monterey, neovim 😬 , npm, vue-cli+Webpack+WebpackDevServer
Production uses Ubuntu 20.04, npm, nginx

* **Vue version:**
Tried both 3.2.22 (my codebase) and 3.1.5 (CodeSandBox forked from your example Sandbox for Calendar)

* **PrimeVue version:** 3.9.1

* **Browser:** Chrome stable latest, Firefox 94.0.1, Safari 15.1 (where CodeSandbox apparently has additional problems...)

**Comment by tugcekucukoglu** [comment]
Accessibility is on PrimeVue [roadmap](https://github.com/primefaces/primevue/wiki/Roadmap#accessibility). For now closing it.
